elephant/spike_train_generation.py

```diff
@@ -204,14 +204,19 @@ def peak_detection(signal, threshold=0.0 * mV, sign='above', format=None):
     """
     assert threshold is not None, "A threshold must be provided"
 
-    if sign is 'above':
+    if sign not in ('above', 'below'):
+        raise AttributeError("sign should be 'above' or 'below'")
+
+    if format is not None and format != 'raw':
+        raise AttributeError("Format argument must be None or 'raw'")
```

Comment:
1. change all asserts and AttributionErros to ValueError
2. did you get AttributeError from our examples styles_guide.rst?
3. better to write `if format not in (None, 'raw')` - the same syntax as you did above;
4. actually, the current version is totally OK, because in the example below the block is small and it's up to a developer to either
```
    if sign is 'above':
        cutout = np.where(signal > threshold)[0]
        peak_func = np.argmax
    elif sign in 'below':
        cutout = np.where(signal < threshold)[0]
        peak_func = np.argmin
    else:
        raise ValueError("sign must be 'above' or 'below'")
```

or

```
if sign not in ('above', 'below'):
        raise ValueError("sign should be 'above' or 'below'")

if sign == 'above':
        cutout = np.where(signal > threshold)[0]	
        peak_func = np.argmax
else:
        cutout = np.where(signal < threshold)[0]
        peak_func = np.argmin
```

The second approach is better in cases where we have long if-blocks.

we should specify this in our guide as well.